src/tests/unit/tests/common/components/cards/how-to-fix-android-card-row.test.tsx

```diff
@@ -0,0 +1,101 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+import { shallow } from 'enzyme';
+import * as React from 'react';
+import { Mock } from 'typemoq';
+
+import { HowToFixAndroidCardRow, HowToFixAndroidCardRowProps } from 'common/components/cards/how-to-fix-android-card-row';
+import { FixInstructionProcessor } from '../../../../../../injected/fix-instruction-processor';
+
+describe(HowToFixAndroidCardRow, () => {
+    it('renders with matches', () => {
+        const fixInstructionProcessorMock = Mock.ofType(FixInstructionProcessor);
+        const props: HowToFixAndroidCardRowProps = {
+            deps: { fixInstructionProcessor: fixInstructionProcessorMock.object },
```

Comment:
`fixInstructionProcessor` is not really been used on `HowToFixAndroidCardRow`... can't it be null?

Reply:
i will be sending another pr to do the color processing

Reply:
added support for color processing now